.pre-commit-config.yaml

```diff
@@ -11,44 +11,53 @@ exclude: >
         ^src/.*/locale/|
         ^src/sentry/data/
     )
+
 repos:
+# For some tools, we install them locally (in the virtualenv).
+# This makes it easier to use them directly (as opposed to pre-commit run),
+# and it's also faster as pre-commit doesn't have to manage them.
+# The thing we lose here is autoupdate, but we have git hooks that
+# let you know when requirements-pre-commit.txt changes.
 -   repo: local
     hooks:
     # Configuration for black exists in pyproject.toml,
     # but we let pre-commit take care of the file filtering.
     - id: black
       name: black
       entry: black
-      language: python
+      language: system
```

Comment:
If I understand correctly, this means that we're in charge to make sure it's installed. Did I understand it correctly?

Reply:
Yeah.